runtime/interpreter/interpreter.go

```diff
@@ -3634,6 +3634,43 @@ func (interpreter *Interpreter) IsSubTypeOfSemaType(subType StaticType, superTyp
 	return sema.IsSubType(semaType, superType)
 }
 
+func (interpreter *Interpreter) domainIterator(address common.Address, domain common.PathDomain) []Value {
```

Comment:
Maybe rename to `domainPaths`, as the function returns an array of paths, not an iterator